Views/MenuWidget.cshtml

```diff
@@ -61,8 +61,10 @@
                     }
                     else
                     {
-                        var activeClass = Context.Request.Path == menuItem.Href?.Partition("?").Left ? "active" : string.Empty;
-                        <li class="nav-item">
+                        var isActive = Context.Request.Path == menuItem.Href?.Partition("?").Left;
+                        var activeClass = isActive ? "active" : string.Empty;

```

Comment:
Don't you have an extension method for these type of conditions?

Reply:
I don't think so. I have an extension where you select the string or the first non-empty alternative from two or more options (OrIfEmpty). Are you thinking of that?

Reply:
Yes.

Reply:
I wrote `If()` for these scenarios: https://github.com/Lombiq/Helpful-Libraries/blob/dev-orchard-1/Libraries/Utilities/StringExtensions.cs#L22

Reply:
I don't really see the advantage, the vanilla is easy enough. Also your method returns null so actually it would be
```csharp
var activeClass = "active".If(isActive) ?? string.Empty;
```

Reply:
The `null` is actually not a problem when rendering the view. It'll render as empty string, too.

Reply:
For me the big advantage is when using this syntax inline, without any intermediary variable.

Reply:
Well it can be inline anyway: `@(isActive ? "active" : string.Empty)`

Reply:
Sure :-) I still find `@("active".If(isActive))` appealing 😉